### PR TITLE
Sentry

### PR DESCRIPTION
**Pull request by SamDepardieu**
# Description

Add Sentry to the scraper

## Type of change

Please delete options that are not relevant.

- [ ] :sparkles: New feature

### CODE REVIEW

File: wsf_scraping/spiders/who_iris_spider.py [hunk 1]
```diff
@@ -15,17 +14,6 @@ class WhoIrisSpider(BaseSpider):
...
-        settings = get_project_settings()
-        years_list = kwargs.get('years_list', False)
-        id = kwargs.get('uuid', '')
-
-        self.uuid = id
-        if years_list:
-            self.years = years_list.split(',')
-        else:
-            self.years = settings['WHO_IRIS_YEARS']
-
```
Comment (hblanks) [inline wsf_scraping/spiders/who_iris_spider.py]: I wasn't sure from the commit message whether this change belonged in the PR or no. Does it?

File: Makefile [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 IMAGE := uk.ac.wellcome/web-scraper
 ECR_IMAGE := 160358319781.dkr.ecr.eu-west-1.amazonaws.com/$(IMAGE)
-VERSION := 2018.10.0
+VERSION := 2018.12.0
```
Comment (nsorros) [inline Makefile]: what happened to 11?

File: wsf_scraping/middlewares.py [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+from twisted.python import log
 from scrapy import signals
 from scrapy import logformatter
 
 
+sentry_sdk.init(os.getenv('SENTRY_DSN'))
+
+
+def log_to_sentry(event):
+    if not event.get('isError') or 'failure' not in event:
```
Comment (nsorros) [inline wsf_scraping/middlewares.py]: isn't that enough `'failure' not in event`

File: Makefile (hunk 2 above)
Comment (hblanks) [reply Makefile]: Good question - The convention we're following here is {YYYY}.{MM}.{SERIAL_NO} - we made releases in October and December but not November.

Comment (nsorros) [reply Makefile]: 👍

File: wsf_scraping/spiders/who_iris_spider.py (hunk 1 above)
Comment (SamDepardieu) [reply wsf_scraping/spiders/who_iris_spider.py]: It doesn't, I just removed this to check if we were still using it. We don't. So we can leave it here and just merge it, or I can open another PR if you prefer

**Pull request merged**